_posts/2020-01-07-jump_dev_2020.md

```diff
@@ -16,31 +16,22 @@ with interests in the software aspects of [JuMP](https://github.com/JuliaOpt/JuM
 related packages. The workshop is open for anybody to attend. In particular, we invite new
 contributors and those who have not met the core development team.
 
+## COVID-19
+
+At present, we are still planning for JuMP-dev to proceed as usual. However, we
+are aware that some institutions have placed travel bans until mid- to late-May.
+
+If these restrictions are extended, or the situation in Belgium changes, we will
+have no option but to cancel. Therefore, we recommend you delay travel planning
+until early May, or make travel plans with refundable fares.
+
+We will update this webpage as information comes to hand.
+
 ## Abstract Submission
 
 **Abstract submission closed on February 29. We will notify accepted speakers in
 early March.**
 
-Similarly to previous iterations of the workshop, we invite participants to
```

Comment:
The call for abstracts still has useful information, e.g., the intended scope of the talks. People might make a decision to attend the workshop based on this.